tests/testsuite.src/used_binaries.at

```diff
@@ -918,3 +912,100 @@ AT_CHECK([cat prog.cob | $COMPILE_MODULE -j -], [0], [job], [],
+       WORKING-STORAGE  SECTION.
+       01 TEST-VAR PIC 9(2) VALUE 'A'.
+       COPY 'CRUD.CPY'.
+       PROCEDURE        DIVISION.
+           DISPLAY TEST-VAR NO ADVANCING
+           END-DISPLAY
+           MOVE 12 TO TEST-VAR
+           DISPLAY TEST-VAR NO ADVANCING
+           END-DISPLAY
+           STOP RUN.
+])
+AT_CHECK([$COBC -Wall -fsyntax-only prog.cob], [1], [], [prog.cob:7: error: CRUD.CPY: No such file or directory
+prog.cob:6: warning: numeric value is expected @<:@-Wothers@:>@
+])
+AT_CHECK([$COMPILE -j -fdiagnostics-show-caret prog.cob], [1], [], [prog.cob:7: error: CRUD.CPY: No such file or directory
+
+  0005          WORKING-STORAGE  SECTION.
+  0006          01 TEST-VAR PIC 9(2) VALUE 'A'.
+  0007 >        COPY 'CRUD.CPY'.
+  0008          PROCEDURE        DIVISION.
+  0009              DISPLAY TEST-VAR NO ADVANCING
+
+prog.cob:6: warning: numeric value is expected
+
+  0004          DATA             DIVISION.
+  0005          WORKING-STORAGE  SECTION.
+  0006 >        01 TEST-VAR PIC 9(2) VALUE 'A'.
+  0007          COPY 'CRUD.CPY'.
+  0008          PROCEDURE        DIVISION.
+
+])
+AT_CHECK([COBC_DIAGNOSTICS_SHOW_CARET=0 $COMPILE -j -fdiagnostics-show-caret prog.cob], [1], [], [prog.cob:7: error: CRUD.CPY: No such file or directory
+
+  0005          WORKING-STORAGE  SECTION.
+  0006          01 TEST-VAR PIC 9(2) VALUE 'A'.
+  0007 >        COPY 'CRUD.CPY'.
+  0008          PROCEDURE        DIVISION.
+  0009              DISPLAY TEST-VAR NO ADVANCING
+
+prog.cob:6: warning: numeric value is expected
+
+  0004          DATA             DIVISION.
+  0005          WORKING-STORAGE  SECTION.
+  0006 >        01 TEST-VAR PIC 9(2) VALUE 'A'.
+  0007          COPY 'CRUD.CPY'.
+  0008          PROCEDURE        DIVISION.
+
+])
+AT_CHECK([COBC_DIAGNOSTICS_SHOW_CARET=1 $COMPILE -j prog.cob], [1], [], [prog.cob:7: error: CRUD.CPY: No such file or directory
+
+  0005          WORKING-STORAGE  SECTION.
+  0006          01 TEST-VAR PIC 9(2) VALUE 'A'.
+  0007 >        COPY 'CRUD.CPY'.
+  0008          PROCEDURE        DIVISION.
+  0009              DISPLAY TEST-VAR NO ADVANCING
+
+prog.cob:6: warning: numeric value is expected
+
+  0004          DATA             DIVISION.
+  0005          WORKING-STORAGE  SECTION.
+  0006 >        01 TEST-VAR PIC 9(2) VALUE 'A'.
+  0007          COPY 'CRUD.CPY'.
+  0008          PROCEDURE        DIVISION.
+
+])
+AT_CHECK([COBC_DIAGNOSTICS_SHOW_CARET=E $COMPILE -j prog.cob], [1], [], [prog.cob:7: error: CRUD.CPY: No such file or directory
+
+  0005          WORKING-STORAGE  SECTION.
+  0006          01 TEST-VAR PIC 9(2) VALUE 'A'.
+  0007 >        COPY 'CRUD.CPY'.
+  0008          PROCEDURE        DIVISION.
+  0009              DISPLAY TEST-VAR NO ADVANCING
+
+prog.cob:6: warning: numeric value is expected
+])
+AT_CHECK([$COMPILE -fdiagnostics-show-caret -fno-diagnostics-show-caret-on-warning -j prog.cob], [1], [], [prog.cob:7: error: CRUD.CPY: No such file or directory
```

Comment:
`-fno-diagnostics-show-caret-on-warning` doesn't exists any more, please replace it with a test for `-fno-diagnostics-show-line-numbers`

Reply:
done, replaced with a test on `-fno-diagnositcs-show-line-numbers`